src/plugins/kibana/server/lib/ingest_build_request.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('lodash');
+const processorTypes = require('../../common/ingest_processor_types');
+
+export default function ingestBuildRequest(pipeline) {
```

Comment:
The name of this file and function is potentially confusing since it's used for the simulate API, and there's an existing Ingest API that's for the actual creation of the index pattern, index template, and pipeline.